src/utils/config.ts

```diff
@@ -30,70 +33,73 @@ export async function getConfiguration(pathToFile) {
  * If it is still unable to find it, it throws an error.
  * @param {string} targetName - the name of the target in question.
  * @param {boolean} viyaSpecific - will fall back to the first target of type SASVIYA.
+ * @returns {Target} target or fallback when one is found.
  */
 export async function findTargetInConfiguration(
-  targetName,
+  targetName: string,
   viyaSpecific = false
-) {
-  const projectRoot = await getProjectRoot()
+): Promise<{ target: Target; isLocal: boolean }> {
```

Comment:
this function doesn't return a Promise

Reply:
`async/await` in ES6/TypeScript is based on `Promise`s, so this function _does_ actually return a `Promise`.